packages/nectar/src/sagas/appConnection/connection.selectors.ts

```diff
@@ -16,7 +18,37 @@ export const initializedRegistrars = createSelector(
   (reducerState) => reducerState.initializedRegistrars
 )
 
+export const connectedPeers = createSelector(
+  connectionSlice,
+  (reducerState) => {
+    return selectedPeersAdapter.getSelectors().selectAll(reducerState.connectedPeers)
+  }
+)
+
+export const connectedPeersMapping = createSelector(
+  certificatesMapping,
+  connectedPeers,
+  (certificates, peers) => {
+    const usersData = Object.values(certificates)
+
+    const usersDataPerPeerId = peers.reduce((accumulator, currentValue) => {
```

Comment:
can you clarify the _accumulator_ as _peersMapping_ and _currentValue_ as _peerId_?